README.md

```diff
@@ -693,6 +693,17 @@ Specifies the number of seconds an IP address is in cooldown after pod deletion.
 **Note:** 0 is a supported value, however it is highly discouraged.
 **Note:** Higher cooldown periods may lead to a higher number of EC2 API calls as IPs are in cooldown cache.
 
+----
+
+#### `DISABLE_CONTAINER_V6` (v1.15.0+)
```

Comment:
nit: `DISABLE_CONTAINER_V6` --> `DISABLE_POD_V6`. We're disabling it at Pod level and so for all the containers in the Pod. Some might misinterpret it.